.github/workflows/talawabot.yml

```
@@ -0,0 +1,25 @@
+
+on:
+  pull_request_review:
+  issue_comment:
+  pull_request:
+
+permissions:
+  issues: write
+  pull-requests: write
+  checks: write
+  statuses: write
+  check_suite:
```

actionlint (1.7.9)

unknown permission scope "check_suite". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)